jetty-core/jetty-tests/jetty-test-client-transports/src/test/java/org/eclipse/jetty/test/client/transport/HttpClientStreamTest.java

```diff
@@ -1189,9 +1188,12 @@ public boolean handle(Request request, org.eclipse.jetty.server.Response respons
             assertTrue(read > 0);
             totalRead += read;
         }
+        assertEquals(64, totalRead);
 
         context.response().write(true, BufferUtil.EMPTY_BUFFER, context.callback());
 
+        assertEquals(-1, stream.read(buffer));
+
         Response response = listener.get(5, TimeUnit.SECONDS);
```

Comment:
Please remove the assert for -1 and replaced it with `listener.await()` rather than `listener.get()`.